CondCore/ESSources/interface/ProductResolver.h

```diff
@@ -158,30 +155,30 @@ class DataProxyWrapper : public cond::DataProxyWrapperBase {
     // multiple IOVs to run concurrently.
     if (m_proxies.size() != nConcurrentIOVs) {
       assert(m_proxies.size() == 1);
-      auto queue = m_edmProxies.front()->queue();
-      auto mutex = m_edmProxies.front()->mutex();
+      auto queue = m_esResolvers.front()->queue();
+      auto mutex = m_esResolvers.front()->mutex();
       for (unsigned int i = 1; i < nConcurrentIOVs; ++i) {
         m_proxies.push_back(std::make_shared<cond::persistency::PayloadProxy<DataT>>(
             &currentIov(), &session(), &requests(), m_source.empty() ? (const char*)nullptr : m_source.c_str()));
-        m_edmProxies.push_back(std::make_shared<DataProxy>(m_proxies[i], queue, mutex));
+        m_esResolvers.push_back(std::make_shared<ProductResolver>(m_proxies[i], queue, mutex));
         // This does nothing except in the special case of a KeyList PayloadProxy.
         // They all need to have copies of the same IOVProxy object.
         m_proxies[i]->initKeyList(*m_proxies[0]);
       }
       assert(m_proxies.size() == nConcurrentIOVs);
     }
-    assert(m_proxies.size() == m_edmProxies.size());
+    assert(m_proxies.size() == m_esResolvers.size());
   }
 
   edm::eventsetup::TypeTag type() const override { return m_type; }
   ProxyP proxy(unsigned int iovIndex) const override { return m_proxies.at(iovIndex); }
-  edmProxyP edmProxy(unsigned int iovIndex) const override { return m_edmProxies.at(iovIndex); }
+  esResolverP esResolver(unsigned int iovIndex) const override { return m_esResolvers.at(iovIndex); }
 
 private:
   std::string m_source;
   edm::eventsetup::TypeTag m_type;
   std::vector<std::shared_ptr<cond::persistency::PayloadProxy<DataT>>> m_proxies;
-  std::vector<std::shared_ptr<DataProxy>> m_edmProxies;
+  std::vector<std::shared_ptr<ProductResolver>> m_esResolvers;
 };
 
 #endif /* CONDCORE_PLUGINSYSTEM_DATAPROXY_H */
```

Comment:
Just for consistency I would change this to:
```suggestion
#endif /* CONDCORE_PLUGINSYSTEM_PRODUCTRESOLVER_H */
```